packages/fast-components-react-msft/src/flyout/flyout.tsx

```diff
@@ -0,0 +1,201 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { get } from "lodash-es";
+import { canUseDOM } from "exenv-es6";
+import { KeyCodes } from "@microsoft/fast-web-utilities";
+import Foundation, { HandledProps } from "@microsoft/fast-components-foundation-react";
+import {
+    AxisPositioningMode,
+    ViewportPositioner,
+    ViewportPositionerClassNameContract,
+    ViewportPositionerHorizontalPosition,
+    ViewportPositionerVerticalPosition,
+} from "@microsoft/fast-components-react-base";
+import {
+    FlyoutAxisPositioningMode,
+    FlyoutHandledProps,
+    FlyoutHorizontalPosition,
+    FlyoutManagedClasses,
+    FlyoutProps,
+    FlyoutUnhandledProps,
+    FlyoutVerticalPosition,
+} from "./flyout.props";
+import {
+    FlyoutClassNameContract,
+    ManagedClasses,
+} from "@microsoft/fast-components-class-name-contracts-msft";
+import { DisplayNamePrefix } from "../utilities";
+
+class Flyout extends Foundation<FlyoutHandledProps, FlyoutUnhandledProps, {}> {
+    public static displayName: string = `${DisplayNamePrefix}Flyout`;
+
+    public static defaultProps: Partial<FlyoutProps> = {
+        disabled: true,
+        horizontalPositioningMode: FlyoutAxisPositioningMode.uncontrolled,
+        defaultHorizontalPosition: FlyoutHorizontalPosition.uncontrolled,
+        verticalPositioningMode: FlyoutAxisPositioningMode.adjacent,
+        defaultVerticalPosition: FlyoutVerticalPosition.bottom,
+        horizontalAlwaysInView: false,
+        verticalAlwaysInView: false,
+        fixedAfterInitialPlacement: false,
+        height: "128px",
+        width: "280px",
+    };
+
+    protected handledProps: HandledProps<FlyoutHandledProps> = {
```

Comment:
if you change this to `HandledProps<Required<FlyoutHandledProps>>` we won't run into the case where this doesn't cause a build error if optional props aren't in `handledProps`